apps.toml

```diff
@@ -3143,6 +3143,14 @@ state = "working"
 subtags = [ "network" ]
 url = "https://github.com/YunoHost-Apps/pihole_ynh"
 
+[pinkarrows]
+added_date = 1726530459 # 2024/09/16
```

Comment:
`added date` and `level` will be added automatically

Reply:
ok i removed them